systemd.d/syslog-ng.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=System Logger Daemon
+Documentation=man:syslog-ng(8)
+Wants=network.target network-online.target
+After=network.target network-online.target
+
+[Service]
+Type=notify
+ExecStart=/usr/sbin/syslog-ng -F $SYSLOGNG_OPTS -p /var/run/syslogd.pid
+ExecReload=/bin/kill -HUP $MAINPID
+EnvironmentFile=-/etc/default/syslog-ng
+EnvironmentFile=-/etc/sysconfig/syslog-ng
```

Comment:
This line should point to /etc/config.d/syslog-ng (lunar does not use sysconfig).